io/disk/Avago_storage_adapter/avago9361_vd.py

```diff
@@ -64,7 +69,8 @@ def setUp(self):
         elif self.raid_level == 'r50' or self.raid_level == 'r60':
             self.pdperarray = 3
         if self.raid_level == 'r0':
-            if 'cc' in str(self.name) or 'offline' in str(self.name):
+            if 'cc' in str(self.name) or 'offline' in str(self.name) or \
+               'rebuild' in str(self.name):
```

Comment:
Also `if str(self.name) in ['cc', 'offline', 'rebuild']:` can be used if you check against them.